test/unit/android_test.rb

```diff
@@ -6,92 +6,91 @@ class AndroidTest < Minitest::Test
   end
 
   test "detect android cupcake (1.5)" do
-    @browser.ua = $ua['ANDROID_CUPCAKE']
-    assert @browser.android?
-    assert @browser.android?(1.5)
+    @browser.ua = Browser['ANDROID_CUPCAKE']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(1.5)
   end
 
   test "detect android donut (1.6)" do
-    @browser.ua = $ua['ANDROID_DONUT']
-    assert @browser.android?
-    assert @browser.android?(1.6)
+    @browser.ua = Browser['ANDROID_DONUT']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(1.6)
   end
 
   test "detect android eclair (2.1)" do
-    @browser.ua = $ua['ANDROID_ECLAIR_21']
-    assert @browser.android?
-    assert @browser.android?(2.1)
+    @browser.ua = Browser['ANDROID_ECLAIR_21']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(2.1)
   end
 
   test "detect android froyo (2.2)" do
-    @browser.ua = $ua['ANDROID_FROYO']
-    assert @browser.android?
-    assert @browser.android?(2.2)
+    @browser.ua = Browser['ANDROID_FROYO']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.